packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { execSync } = require( 'child_process' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+const path = require( 'path' );
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*` directories to the latest version.
+ * Changes in `packages/*` will be committed as well.
+ *
+ * See https://github.com/cksource/ckeditor5-internal/issues/1123
+ *
+ * @param {Object} options
+ * @param {String} options.cwd Current working directory (packages) from which all paths will be resolved.
+ * @param {Boolean} options.dryRun If set to true, commit of changes to the dependencies in `packages/*` will be reversed.
+ *
+ * @returns {Promise}
+ */
+module.exports = function updatePackageVersions( options ) {
+	const pathToPackages = path.posix.join( options.cwd, 'packages' );
+	const pathToRelease = path.posix.join( options.cwd, 'release' );
+
+	const execOptions = {
+		stdio: 'inherit',
+		cwd: options.cwd
+	};
+
+	try {
+		console.log( '\n📍 ' + chalk.blue( 'Updating \'packages/*\'...' ) );
+		const packagesChanged = updateDirectory( pathToPackages );
+
+		if ( packagesChanged ) {
+			console.log( '\n📍 ' + chalk.blue( 'Committing the changes...\n' ) );
+
+			execSync( 'git add -A', execOptions );
```

Comment:
If the `dryRun` flag is set, no changes should be commited. Maybe something like here https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-dev-env/lib/release-tools/tasks/bumpversions.js#L217-L228 could be used instead: list the changes that would be commited if `dryRun` flag was not set.

Besides that, the `git add -A` operates on the entire working tree, and not just on the current path so I think it should not be used here. It would be better to just stage the files that are modified by this script only.